pyvista/plotting/plotter.py

```diff
@@ -2981,60 +3028,60 @@ def add_composite(
-        show_scalar_bar=None,
-        multi_colors=False,
-        name=None,
-        texture=None,
-        render_points_as_spheres=None,
-        render_lines_as_tubes=None,
-        smooth_shading=None,
-        split_sharp_edges=None,
-        ambient=None,
-        diffuse=None,
-        specular=None,
-        specular_power=None,
-        nan_color=None,
-        nan_opacity=1.0,
-        culling=None,
-        rgb=None,
-        categories=False,
-        silhouette=None,
-        use_transparency=False,
-        below_color=None,
-        above_color=None,
-        annotations=None,
-        pickable=True,
-        preference='point',
-        log_scale=False,
-        pbr=None,
-        metallic=None,
-        roughness=None,
-        render=True,
-        user_matrix=None,
-        component=None,
-        emissive=None,
-        copy_mesh=False,
-        backface_params=None,
-        show_vertices=None,
-        edge_opacity=None,
+        mesh: NumpyArray[float] | pyvista.DataSet | pyvista.MultiBlock | _vtk.vtkAlgorithm,
+        color: ColorLike | None = None,
+        style: Literal['surface', 'wireframe', 'points', 'points_gaussian'] | None = None,
+        scalars: str | NumpyArray[float] | None = None,
+        clim: Sequence[float] | None = None,
+        show_edges: bool | None = None,
+        edge_color: ColorLike | None = None,
+        point_size: float | None = None,
+        line_width: float | None = None,
+        opacity: float | OpacityOptions | Sequence[float] | None = None,
+        flip_scalars: bool = False,
+        lighting: bool | None = None,
+        n_colors: int = 256,
+        interpolate_before_map: bool | None = None,
+        cmap: str | list[str] | pyvista.LookupTable | None = None,
+        label: str | None = None,
+        reset_camera: bool | None = None,
+        scalar_bar_args: ScalarBarArgs | None = None,
+        show_scalar_bar: bool | None = None,
+        multi_colors: bool = False,
+        name: str | None = None,
+        texture: pyvista.Texture | NumpyArray[float] | None = None,
+        render_points_as_spheres: bool | None = None,
+        render_lines_as_tubes: bool | None = None,
+        smooth_shading: bool | None = None,
+        split_sharp_edges: bool | None = None,
+        ambient: float | None = None,
+        diffuse: float | None = None,
+        specular: float | None = None,
+        specular_power: float | None = None,
+        nan_color: ColorLike | None = None,
+        nan_opacity: float = 1.0,
+        culling: Literal['front', 'back'] | None = None,
+        rgb: bool | None = None,
+        categories: bool = False,
+        silhouette: Silhouette | bool | None = None,
+        use_transparency: bool = False,
+        below_color: ColorLike | None = None,
+        above_color: ColorLike | None = None,
+        annotations: dict[float, str] | None = None,
+        pickable: bool = True,
+        preference: Literal['point', 'cell'] = 'point',
```

Comment:
import from `core.utilities.arrays`
```suggestion
        preference: PointLiteral | CellLiteral = 'point',
```